_data/sites.json

```diff
@@ -1424,7 +1424,14 @@
             "dailymile.com"
         ]
     },
-
+    {
```

Comment:
There should be empty lines separating the entry from the ones before and after it.